community/kernel/src/main/java/org/neo4j/kernel/impl/api/cursor/TxSingleRelationshipCursor.java

```diff
@@ -43,16 +47,14 @@ public TxSingleRelationshipCursor init( RelationshipCursor cursor )
     @Override
     public boolean next()
     {
-        boolean exists = cursor.next();
-
-        long id = cursor.getId();
-
         if ( state.relationshipIsDeletedInThisTx( id ) )
         {
             visit( -1, -1, -1, -1 );
```

Comment:
it should preferably be several constants, a different one for each of these fields.